macos/library/FLEFileChooserPlugin.m

```diff
@@ -30,11 +30,20 @@
 static NSString *const kPlatformMethodNameKey = @"method";
 static NSString *const kPlatformMethodArgsKey = @"args";
 
-// Method argument keys.
+// Appearance configuration keys for NSSavePanel.
+static NSString *const kPanelTitleKey = @"title";
+static NSString *const kPanelMessageKey = @"message";
```

Comment:
Per offline discussion, since NSSavePanel's title no longer shows up in 10.13, and I've never encountered a save/open panel using message (and it looks bad on 10.13, which is more evidence that it's extremely rare), let's remove these. They also aren't easily implementable on Linux.

Reply:
Done.